src/instructlab/model/download.py

```diff
@@ -109,7 +111,7 @@ def download_gguf(self) -> None:
             )
             raise click.exceptions.Exit(1)
 
-    def download_safetensors(self) -> None:
+    def download_safetensors_or_bin(self) -> None:
```

Comment:
minor minor minor nit: I prefer calling it something along the lines of `download_entire_hf_repo` or just something that indicates that's what it's doing

the logic inside `download` decides when to download the whole repo (i.e. safe tensors or bin) and then just calls the function that downloads the whole repo

Reply:
renamed to `download_entire_hf_repo`